conftest.py

```diff
@@ -30,7 +29,7 @@ def clean_parso_cache():
     """
     old = cache._default_cache_path
     tmp = tempfile.mkdtemp(prefix='parso-test-')
-    cache._default_cache_path = tmp
+    cache._default_cache_path = Path(str(tmp))
```

Comment:
Do we need this `str()` call here? AFAIK `tempfile.mkdtemp` always returns a string. If you wish to ensure, I guess `os.fspath` would be more convenient but as I said I am unsure whether needed or not.

Reply:
Did not know this. Changed.